3-extra/1-radio-stations.js

```diff
@@ -25,6 +34,16 @@
  * - Return only the frequencies that are radio stations.
  */
 // `getStations` goes here
+function getStations () {
```

Comment:
Nice solution 👍 
For practice, can you re-write this using the `filter` array method?

Reply:
done :)